docker-compose.yml

```diff
@@ -17,13 +17,15 @@ services:
     volumes:
       - "${PWD}:/app"
       - tox-data:/app/.tox
-    command: gunicorn pyslackersweb:app_factory --access-logfile - --bind=0.0.0.0:8000 --worker-class=aiohttp.GunicornWebWorker --reload
+    command: "/bin/sh -c 'PYTHONPATH=. alembic upgrade head && gunicorn pyslackersweb:app_factory --access-logfile - --bind=0.0.0.0:8000 --worker-class=aiohttp.GunicornWebWorker --reload'"
 
   redis:
     image: redis:5-alpine
 
   postgresql:
     image: postgres:11
+    ports:
```

Comment:
If redis isn’t exposed, why expose the database?

Reply:
This way it's easier to inspect the database with an external tool, datagrip, dbeaver, ...